scripts/basic-workflow-tests.sh

```diff
@@ -186,16 +186,16 @@ test_serve() {
     serve_args=()
     if [ "${model_type}" == "base" ]; then
         if [ "${BACKEND}" == "vllm" ]; then
-            model_path="${CACHE_HOME}/instructlab/models/${GRANITE_SAFETENSOR_REPO}"
+            model_path="${CACHE_HOME}/instructlab/models/${GRANITE_SAFETENSOR_REPO}/main/"
         else    # if not using vLLM, use the GGUF for llama-cpp
-            model_path="${CACHE_HOME}/instructlab/models/${GRANITE_GGUF_MODEL}"
+            model_path="${CACHE_HOME}/instructlab/models/instructlab/granite-7b-lab-GGUF/main/${GRANITE_GGUF_MODEL}"
         fi
     elif [ "$model_type" == "teacher" ]; then
         if [ "${MIXTRAL}" -eq 1 ]; then
-            model_path="${CACHE_HOME}/instructlab/models/${MIXTRAL_GGUF_MODEL}"
-            serve_args+=("--model-family" "mixtral")
+            model_path="${CACHE_HOME}/instructlab/models/TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF/main/${MIXTRAL_GGUF_MODEL}"
+            SERVE_ARGS+=("--model-family" "mixtral")
```

Comment:
Why do `s/serve_args/SERVE_ARGS/` and not the same for `model_path`?

Reply:
un-did the change, dont remember how that happened
thanks!